nncf/graph/operator_metatypes.py

```diff
@@ -125,13 +126,13 @@ def matches(cls, module_attributes: Optional[ModuleAttributes] = None,
 @PT_OPERATOR_METATYPES.register()
 class InputNoopMetatype(PTOperatorMetatype):
     name = "input_noop"
-    external_op_names = [name, MODEL_INPUT_OP_NAME]
+    external_op_names = [name, MODEL_INPUT_OP_NAME, NNCFGraphNodeType.INPUT_NODE]
 
 
 @PT_OPERATOR_METATYPES.register()
 class OutputNoopMetatype(PTOperatorMetatype):
     name = "output_noop"
-    external_op_names = [name, MODEL_OUTPUT_OP_NAME]
+    external_op_names = [name, MODEL_OUTPUT_OP_NAME, NNCFGraphNodeType.OUTPUT_NODE]
```

Comment:
Should we really have two names for the same node type?

Reply:
Yes, we should must have two names here, because the nodes are matched by operator too.